src/api.rs

```diff
@@ -63,8 +65,8 @@ impl Bed {
 impl BedBuilder {
     pub fn new<P: AsRef<Path>>(path: P) -> Self {
         Self {
-            path: Some(PathBuf::from(path.as_ref())),
-            count_a1: Some(true),
+            path: Some(path.as_ref().into()),
+            count_a1: None,
```

Comment:
Now that the derived build method is being invoked via `build_no_file_check` it doesn't make sense to declare the default for this field here in addition to declaring it in the attribute above.